source/standards/programming-languages.html.md.erb

```diff
@@ -98,6 +98,24 @@ We used Scala in the early days of GDS. GOV.UK Licensing is the only remaining
 application written in Scala but we’ve found it hard to support because of a lack
 of skills in GDS. Do not use Scala for new projects.
 
+## Mobile Development
+
+For developing mobile apps, we use Swift for iOS and Kotlin for Android.
+
+These languages have been chosen over using frameworks such as Flutter or .NET MAUI so that the apps
+feel like a native Android/iOS app without writing extra code to account for the specific OS in Flutter, for example. 
+
+### Swift
+
+The Swift version is generally tied to the version of Xcode you're using. As long as the Xcode version supports the minimum targeted iOS version 
+of your team's app, you can use the most recent version if desired. 
```

Comment:
huh the suggestion tool disappeared from github, I must be doing something odd...

Consider putting this on one line